packages/amplify-graphql-schema-generator/src/ts-schema-generator/helpers.ts

```diff
@@ -190,33 +195,35 @@ export const createSchema = (schema: Schema, config?: DataSourceGenerateConfig):
 
   const engine = schema.getEngine().type;
 
-  const nullEnums = schema  // find null enums in all models
+  const nullEnumFields = schema // find null enum fields in all models
     .getModels()
-    .map((model) => model
+    .map((model) =>
+      model
         .getFields()
         .filter((field) => field.type.kind === 'Enum')
         .map((field) => {
-          if(field.type.kind === 'Enum'){
+          if (field.type.kind === 'Enum') {
             return createEnums(field.type, engine, model.getName(), field.name);
-          }
-          else{
+          } else {
             return undefined;
           }
-     }));
+        }),
+    );
 
-    const notNullEnums = schema  // find Non null enums in all models
+  const RequiredEnumFields = schema // find required enum fields in all models
```

Comment:
nit: don't PascalCase non-type, non-Class variables.
```suggestion
  const requiredEnumFields = schema
```

Reply:
Done